assets/js/pages/Profile/ProfileForm.jsx

```diff
@@ -25,6 +25,7 @@ function ProfileForm({
   disableForm,
   passwordModalOpen = false,
   totpBoxOpen = false,
+  singleSignOnEnabled = false,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** No need for default value when is `false`‏
```suggestion
  singleSignOnEnabled,
```

Reply:
Thank you @balanza 
Yes, good tip. Let me send my own commit, as there are other `false` defaults on top, and I will change them all, otherwise it will be strange

Reply:
@balanza At the end, I think I'm going to revert this change...